frontend/src/scenes/dashboard/DashboardItem.tsx

```diff
@@ -246,8 +248,8 @@ export function DashboardItem({
         }
 
         // Insight agnostic empty states
-        if (showErrorMessage) {
-            return <InsightErrorState />
+        if (showErrorMessage || receivedErrorFromAPI) {
+            return <InsightErrorState excludeDetail={true} />
```

Comment:
`InsightErrorState`'s gray background makes sense in the larger size (`excludeDetail={false}`) but here it looks kind of sloppy:
<img width="703" alt="144045452-e10f5c2f-fde0-42b5-a589-27f35e86ecb3 (1)" src="https://user-images.githubusercontent.com/4550621/144072909-2f9a42f8-f41e-40f2-8281-3f5fbfd784cb.png">
I'm going to rely on @clarkus's skills to suggest the better way though.

Reply:
These errors states do look a little cramped. If we retain the gray background, we should pad the container a bit more. If it's feasible to drop the gray background and reuse colors from the dashboard item color styles, then we wouldn't need to change padding. Put together a quick example:
![Frame 9192](https://user-images.githubusercontent.com/254612/144084679-ea962139-426f-49ac-83c2-10ee6ffb58d7.png)